src/pages/errors.vue

```diff
@@ -36,49 +40,106 @@ const defaultDialogOptions = {
 }
 
 const showUnauthorizedAccess = ref(dialogToShow.value === ErrorModalsE.UNAUTHORIZED_ACCESS_TO_ENTITY)
+
 const registriesContact = getContactInfo('registries')
 
+const getNrErrorMessage = (nrState: NameRequestStateE): string => {
+  switch (nrState) {
+    case NameRequestStateE.EXPIRED:
+      return 'The name request has expired.'
+    case NameRequestStateE.CONSUMED:
+      return 'The name request has already been consumed.'
+    case NameRequestStateE.NOT_APPROVED:
+      return 'The name request has not been approved.'
+    case NameRequestStateE.NOT_FOUND:
+      return 'The name request number could not be found.'
+    case NameRequestStateE.NEED_CONSENT:
+      return 'The name request number is awaiting consent.'
+    default:
+      return 'An unexpected error has occurred.'
+  }
+}
+
+const nrErrorMessage = invalidNameRequest ? getNrErrorMessage(nrState) : ''
 </script>
 
 <template>
-  <div class="mx-auto my-auto">
-    <UButton
-      variant="link"
-      data-cy="access-denied-retry-dashboard-page-button"
-      @click="goToEntityDashboard()"
-    >
-      Go to manage business
-    </UButton>
-  </div>
   <!--  if we get to have more error modals then 2, move the error modals to their own component files -->
   <BcrosDialogCardedModal
     name="UnauthorizedAccessToEntityErrorModal"
-    :display="showUnauthorizedAccess"
+    :display="showUnauthorizedAccess || !!invalidNameRequest"
     :options="defaultDialogOptions"
     @close="goHome()"
   >
     <template #header>
       <span class="text-2xl font-light">
-        Unable to Access Business
+        <template v-if="showUnauthorizedAccess">
+          {{ isTempReg ? 'Unable to Access Incorporation Application' : 'Unable to Access Business' }}
```

Comment:
A Registration, an Amalgamation, and a Continuation In also use a temporary registration number. You probably need a different message for those.

Or, maybe you can get away with "Unable to Access Registration", but confirm with your UX person.

Reply:
Sample sole prop registration with expired NR:
https://business-dashboard-dev--pr-90-un4amkde.web.app/TFyrdiG0Vg

Still shows "Invalid Incorporation Application":

![image](https://github.com/user-attachments/assets/b82c177f-ce63-428e-b8ac-c007f2766ba3)

Reply:
The header should now show "Invalid Name Request". I just ran gcbrun. It may take some time for the temp url to be updated.

Reply:
Looks good. Resolving.

Reply:
Tests:

![image](https://github.com/user-attachments/assets/79b8b2a6-e306-416d-8fba-4e66d7c06971)

![image](https://github.com/user-attachments/assets/233e9d58-e1f4-4260-b20e-49aca9958ede)

![image](https://github.com/user-attachments/assets/d65ce7aa-8c31-4bda-ab16-5f3ca8250991)

![image](https://github.com/user-attachments/assets/fbfe53bd-8dd8-4174-afde-1900e9c6beff)